File: Modul4/C/PRAK405-2310817210003-MuhammadAzrianzan.c
#include <stdio.h>

int main() {
    int n, m;
    scanf("%d %d", &n, &m);
    int totalHasil=0, hasilBaris=0;

    for (int i = 1; i < n+1; i++) {
        for (int j = i; j > 0; j--) {
            if (j > 1) {
                int hasilTunggal = (j * m);
                hasilBaris = hasilBaris + hasilTunggal;
                printf("(%d*%d) + ", j, m);
            }
            else if (j == 1) {
                int hasilTerakhir = (j * m);
                hasilBaris = hasilBaris + hasilTerakhir;
                printf("(%d*%d) = %d\n", j, m, hasilBaris);
                totalHasil = totalHasil + hasilBaris;
                hasilBaris = 0;
            }
        }
    }
    printf("%d", totalHasil);
    return 0;
}

File: Modul4/Python/PRAK405-2310817210003-MuhammadAzrianzan.py
n, m = input("").split()
n = int(n)
m = int(m)
totalHasil = 0
hasilBaris = 0

for i in range(1, n+1):
    for j in range(i, 0, -1):
        if j > 1:
            hasilTunggal = (j * m)
            hasilBaris = hasilBaris + hasilTunggal
            print(f"({j}*{m}) + ", end="")
        elif j == 1:
            hasilTerakhir = (j * m)
            hasilBaris = hasilBaris + hasilTerakhir
            print(f"({j}*{m}) = {hasilBaris}")
            totalHasil = totalHasil + hasilBaris
            hasilBaris = 0
print(f"{totalHasil}")